src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -2,10 +2,12 @@ import { Box, Flex, Sans } from "@artsy/palette"
 import { ArtworkActions_artwork } from "__generated__/ArtworkActions_artwork.graphql"
 import SearchIcon from "lib/Icons/SearchIcon"
 import React from "react"
-import { View } from "react-native"
+import { NativeModules, View } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components/native"
 
+const Constants = NativeModules.ARCocoaConstantsModule || {}
```

Comment:
I totally borrowed that from the [other place](https://github.com/artsy/emission/blob/master/src/lib/relay/createEnvironment.ts#L10) that references these variables. Will `NativeModules.ARCocoaConstantsModule` always exist?

Reply:
Ah, that makes sense! I _think_ `NativeModules.ARCocoaConstantsModule` should be defined, since:

- the Objective-C code defines it when running in an app context, and
- the Jest setup defines it when running in a testing context.

But I could be missing something – what do you think, @alloy?

Reply:
Yeah it should exist, that other place also doesn’t need the fallback (my bad). If perhaps it was added for tests, then that just means a mock for it is missing in `setupJest.ts`

Reply:
Okay cool, thanks for that context! I'll follow-up with another PR to clean these up 👍

Reply:
Woo! Thanks, team!